spec/Section 5 -- Validation.md

```diff
@@ -429,6 +429,10 @@ SameResponseShape(fieldA, fieldB):
 
   * Let {typeA} be the return type of {fieldA}.
   * Let {typeB} be the return type of {fieldB}.
+  * Let {fieldARequiredStatus} be the required status of {fieldA}
+  * Let {fieldBRequiredStatus} be the required status of {fieldB}
+  * Let {typeA} be the result of {ModifiedOutputType(typeA, fieldARequiredStatus)}
```

Comment:
Am I allowed to reference functions defined in other documents? `ModifiedOutputType` is defined in `Execution.md`

Reply:
Yes! that's okay.

However you should not redefine a variable. In spec-pseudo-programming "Let {x} be" means `const x =`

Reply:
Ah I was pulling this pattern from line 438 in the same file. I'll create a new variable.